UI/window-basic-main.cpp

```diff
@@ -1049,6 +1049,9 @@ bool OBSBasic::InitBasicConfigDefaults()
 	config_set_default_uint  (basicConfig, "AdvOut", "Track5Bitrate", 160);
 	config_set_default_uint  (basicConfig, "AdvOut", "Track6Bitrate", 160);
 
+	config_set_default_bool  (basicConfig, "AdvOut", "RecRB", false);
```

Comment:
It's weird that the checkbox would default to checked since the default value in the config is `false`.